### PR TITLE
Define Item Categories

### PR DESCRIPTION
**Pull request by Mloweedgar**

### CODE REVIEW

File: src/Resources/components/ItemCategories/List/index.js [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+      );
+    });
+
+    this.setState({
+      selectedItemCategories: selectedList,
+      selectedPages: pages,
+    });
+  };
+
+  /**
```
Comment (BenMaruchu) [inline src/Resources/components/ItemCategories/List/index.js:130]: Can you remove this part, since it is not applicable for now

**Review by BenMaruchu — CHANGES_REQUESTED** [review]

File: src/Resources/components/ItemCategories/List/index.js [hunk 2]
```diff
@@ -194,31 +188,28 @@ class ItemCategoriesList extends Component {
...
           onRefresh={() =>
             refreshItemCategories(
               () => {
-                notifySuccess('ItemCategories refreshed successfully');
+                notifySuccess('Item categories refreshed successfully');
               },
               () => {
                 notifyError(
-                  'An Error occurred while refreshing ItemCategories please contact system administrator'
+                  'An Error occurred while refreshing Item categories please contact system administrator'
```
Comment (navish) [inline src/Resources/components/ItemCategories/List/index.js:190]: change case on Item category to item category

File: src/Resources/components/ItemCategories/List/index.js [hunk 3]
```diff
@@ -264,11 +255,11 @@ class ItemCategoriesList extends Component {
                 deleteItemCategory(
                   itemCategory._id, // eslint-disable-line
                   () => {
-                    notifySuccess('Item was archived successfully');
+                    notifySuccess('Item category was archived successfully');
                   },
                   () => {
                     notifyError(
-                      'An Error occurred while archiving Item please contact system administrator'
+                      'An Error occurred while archiving Item category please contact system administrator'
```
Comment (navish) [inline src/Resources/components/ItemCategories/List/index.js]: change case on Item category to item category

**Review by navish — CHANGES_REQUESTED** [review]
Update the cases on commented lines. and cross check any other appearances

**Review by BenMaruchu — APPROVED** [review]

**Review by navish — APPROVED** [review]

**Pull request merged**